addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -319,10 +319,11 @@
     <!-- from it. -->
     <xpath expr="//div[hasclass('o_we_image_options')]" position="replace">
         <div data-selector="span.fa, i.fa, img">
-            <we-select string="Alignment" data-state-to-first-class="true">
-                <we-button data-select-class="float-start" title="Align Left">Left</we-button>
-                <we-button data-select-class="" title="Align Center">Center</we-button>
-                <we-button data-select-class="float-end" title="Align Right">Right</we-button>
+            <we-select string="Alignment" data-state-to-first-class="true" data-no-preview="true">
+                <we-button data-set-alignment="Unalign" title="Unalign">None</we-button>
```

Comment:
@visp-odoo i think it should be named correctly we don't aim to unAlign we are setting alignment to none so it should be `data-set-alignment="none" title="None"` or something better @qsm-odoo what is your view on this? And if Unalign is good then it should camelCased `unAlign`

Reply:
@shsa-odoo I noticed that they have already used the term `Unalign` instead of `none`, so I have followed suit and used `Unalign` instead of `none`.